src/components/signup-view/signup-view.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+
+export const SignupView = () => {
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+    const [email, setEmail] = useState('');
+    const [birthday, setBirthday] = useState('');
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+
+        const data = {
+            Username: username,
+            Password: password,
+            Email: email,
+            Birthday: birthday
+        }
+
+        fetch('https://myflix-88009.herokuapp.com/users', {
+            method: 'POST',
+            body: JSON.stringify(data),
+            headers: {
+                'Content-Type': 'application/json'
+            }
+        }).then((response) => {
+            if (response.ok) {
+                alert('Signup successful');
+                window.location.reload();
+            } else {
+                console.log(data)
+                alert('Signup failed')
+            }
+        });  
+    };
+
+    return (
+        <form onSubmit={handleSubmit}>
+            <label>
+                Username:
+                <input 
+                type='text'
+                value={username}
+                onChange={(e) => setUsername(e.target.value)}required
+                minLength='3'
+                />
+            </label>
+            <label>
+                Password:
+                <input 
+                type='password'
+                value={password}
+                onChange={(e) => setPassword(e.target.value)}required
+                />
+            </label>
+            <label>
+                Email:
+                <input 
+                type='text'
```

Comment:
You could use `type=email` for email validation here, remember the html5 validation?

Reply:
done!Thanks